### PR TITLE
new pullrequest

### PR DESCRIPTION
**Pull request by ujeanBal**

### CODE REVIEW

File: SmartHouse.Entities/SmartHouse.Entities/Device.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        public IList<HouseSlaveInvoker> Methods { get; set; }
+
+        public Device()
+        {
+            Methods = new List<HouseSlaveInvoker>();
+        }
+
+        public Device(IList<HouseSlaveInvoker> methods)
+        {
+            Methods = new List<HouseSlaveInvoker>();
```
Comment (Artem-Marakhovskyi) [inline SmartHouse.Entities/SmartHouse.Entities/Device.cs]: Let one `ctor` call another with `public Device(...) : this()`. Initialization of `Methods` will happen at `ctor` without params

Comment (ujeanBal) [reply SmartHouse.Entities/SmartHouse.Entities/Device.cs]: это я что-то тормазнула

Comment (ujeanBal) [reply SmartHouse.Entities/SmartHouse.Entities/Device.cs]: я вообще удалю второй коснтруктор..это же публичное свойство...

File: SmartHouse.Entities/SmartHouse.Entities/SlaveType.cs [hunk 2]
```diff
@@ -0,0 +1,8 @@
+﻿namespace SmartHouse.Entities
+{
+    public enum SlaveType
+    {
+        Sensor,
```
Comment (KarachentsevDanil) [inline SmartHouse.Entities/SmartHouse.Entities/SlaveType.cs]: Is Sensor is default for enum? Please add Unknown value

File: SmartHouse.Entities/SmartHouse.Entities/SlaveStatus.cs [hunk 3]
```diff
@@ -0,0 +1,9 @@
+﻿namespace SmartHouse.Entities
+{
+    public enum SlaveStatus
+    {
+        On,
```
Comment (KarachentsevDanil) [inline SmartHouse.Entities/SmartHouse.Entities/SlaveStatus.cs]: Is On is default for enum? Please add Unknown property

Comment (Artem-Marakhovskyi) [reply SmartHouse.Entities/SmartHouse.Entities/SlaveStatus.cs]: Decided not to add Unknown in order not to introduce additional logic

File: SmartHouse.Entities/SmartHouse.Entities/SlaveType.cs (hunk 2 above)
Comment (Artem-Marakhovskyi) [reply SmartHouse.Entities/SmartHouse.Entities/SlaveType.cs]: Decided not to add Unknown in order not to introduce additional logic

**Pull request merged**